### PR TITLE
More realistic examples of braces.

### PR DESCRIPTION
**Pull request by rpspringuel**

### CODE REVIEW

**Comment by eroux** [comment]
ok for me

File: illus/ob-main.tex [hunk 1]
```diff
@@ -20,7 +20,13 @@
 \gresetclef{invisible}
 \gresetlyriccentering{syllable}
 \greseteolcustos{manual}
-\hsize=4.1in
-\gabcsnippet[0]{( ) g<v>[</v>ob:1;6mm<v>]</v>!hgh(g[ob:1;6mm]!hgh) g<v>[</v>ocb:1;6mm<v>]</v>!hgh(g[ocb:1;6mm]!hgh) g<v>[</v>ocba:1<v>\{]</v>!hgh<v>[</v>ocba:0<v>\}]</v>(g[ocba:1{]!hgh[ocba:0}]) ()}
+%\gresetinitiallines{0}
+\hsize=5in
+\gabcsnippet[0]{( ) %g(g)
+gr<v>[</v>ob:1;6mm<v>]</v>(gr[ob:1;17.5mm]) gh(gh)
```
Comment (henryso) [inline illus/ob-main.tex]: The text in the score doesn't match the text in the parentheses.

**Comment by rpspringuel** [comment]
Fixed the mismatch.

**Comment by henryso** [comment]
Looks good.

**Pull request merged**